### PR TITLE
vmagent: add "/api/v1/push" to request handler

### PR DESCRIPTION
**Pull request by edma2**
We have deployed vmagent as a proxy in front of Grafana Mimir which uses `/api/v1/push` as the [remote write endpoint](https://grafana.com/docs/mimir/v2.11.x/references/http-api/#remote-write). In order to support this use-case, it'd be great if vmagent could support `/api/v1/push` as an additional supported path. I'm also open to making this a flag but I started with the simpler approach.

### CODE REVIEW

File: app/vmagent/main.go [hunk 1]
```diff
@@ -261,7 +261,7 @@ func requestHandler(w http.ResponseWriter, r *http.Request) bool {
 		path = strings.TrimSuffix(path, "/")
 	}
 	switch path {
-	case "/prometheus/api/v1/write", "/api/v1/write":
+	case "/prometheus/api/v1/write", "/api/v1/write", "/api/v1/push":
```
Comment (hagen1778) [inline app/vmagent/main.go:264]: Seems like `/api/v1/push` could potentially conflict with Loki ingestion path, if we ever will add its support to vmagent.
Adding a prefix `prometheus/api/v1/push` would help though.

Workarounds are the following:
1. Configure pushing clients to use [standard Prometheus path](https://prometheus.io/docs/prometheus/latest/querying/api/#remote-write-receiver) `/api/v1/push`
2. Use vmauth to reroute requests to the correct path.

Comment (hagen1778) [reply app/vmagent/main.go:264]: Makes sense, thanks

**Comment by hagen1778** [comment]
@valyala do you have any objections?

**Comment by valyala** [comment]
@edma2 , thanks for the pull request!

**Comment by valyala** [comment]
FYI, this pull request has been included in [vmagent v1.100.0](https://docs.victoriametrics.com/changelog/).

**Pull request merged**